### DISCUSSION TITLE
Issue #1990 [Bug]: Lazy loaded table results in browser console warnings and errors

### DISCUSSION THREAD

**Discussion by darbyfelton** [rappasoft/laravel-livewire-tables#2063] [General]
Hi @lrljoe, I would like to test the [fixes](https://github.com/rappasoft/laravel-livewire-tables/issues/1990#issuecomment-2466565808) for #1990 if possible.

I have a working branch of my project that is now running v3.5.2, and the reported issue is present.

How might I test the fixes?

Thank you!

**Comment by lrljoe** [comment]
I will create a temporary test branch for this specific issue tomorrow.

**Reply by lrljoe** [reply]
Apologies for the delay, it won't be in the next release, but the one after.

Touch base with me on the official Discord if you want an unofficial interim fix.

**Reply by lrljoe** [reply]
@darbyfelton - the fix is in the "development" branch as of today :)

**Comment by lrljoe** [comment]
Should now be fixed in latest version

**Comment by qwasyx0** [comment]
[Placeholder HTML](https://github.com/rappasoft/laravel-livewire-tables/issues/1990#issuecomment-2571381863) still produces the issue.
